scripts/monitoring/cron-send-docker-metrics.py

```diff
@@ -22,9 +23,11 @@
     zs = ZaggSender()
     try:
         cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
+        cli_storage = Client(base_url='unix://var/run/docker.sock')
```

Comment:
The reason we use `AutoVersionClient` is so that the version of docker can be different than what Dockerpy is hard coded for.

Is there a reason to not use the `AutoVersionClient` that's already here in the `cli` variable?